sdw_util/Util.py

```diff
@@ -148,6 +150,45 @@ def get_qubes_version():
     return version
 
 
+def get_whonix_version():
+    """
+    Obtain Whonix version from /srv/formulas/base/virtual-machine-formula/qvm/whonix.jinja
+
+    In '.sls' files the whonix version is obtained with
+
+        {% import "qvm/whonix.jinja" as whonix %}
+        {{ whonix.whonix_version }}
+
+    This function extracts this value so that it can be on this script.
+    """
+
+    # "caching" because qubesctl takes several seconds
+    global whonix_version  # noqa: PLW0603
```

Comment:
If we need caching, I would suggest using [`functools.cache`](https://docs.python.org/3/library/functools.html#functools.cache), instead of a global variable.